src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -160,8 +160,20 @@ def _load_package_tool(tool_name, module_name, class_name, method_name, node_inp
                 message=f"Required inputs {list(missing_inputs)} are not provided for tool '{tool_name}'.",
                 target=ErrorTarget.EXECUTOR,
             )
+        try:
+            api = getattr(provider_class(**init_inputs_values), method_name)
+        except Exception as ex:
+            raise ToolLoadError(
+                module=module_name,
```

Comment:
What is this used for? Is it required for this exception?

Reply:
Its most important function is to record the error target and module.